resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -5380,6 +5380,934 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Auth Type Callback (Execution) " enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
+          <stringProp name="LoopController.loops">1</stringProp>
+        </elementProp>
+        <stringProp name="ThreadGroup.num_threads">1</stringProp>
+        <stringProp name="ThreadGroup.ramp_time">1</stringProp>
+        <boolProp name="ThreadGroup.scheduler">false</boolProp>
+        <stringProp name="ThreadGroup.duration"></stringProp>
```

Comment:
Download Card Event (Preparation) - might take a little longer time when there are many samples to process, how are taking care of this before execution can run ?

Reply:
We have not run the execution for this API yet, it was working for few of the samples so have added the API in the PR.

Reply:
We have changed the preparation API. We are now using Request Card VID API.

Reply:
After the change we have test this and there are no errors?

Reply:
Yes, with this preparation we have checked. Also, I have added a constant timer to provide some delay. As it takes a bit of time for the card to be in ready to download status.

Reply:
I saw the default, but 30 sec will be long since if we want 10k samples, we will be looking at 8+ hours with 10 users. Can we not keep the timer as a tire down thead group ? Also variable needs to named more appropriately

Reply:
Have updated the variable name as "downloadCardDelay", will this be okay ? 
And we can decrease the delay time from 30 to 15 or 10 sec. 
Why need to add a separate tear down thread ?
We can also add delay directly provide in the start up delay option present in thread group.

Reply:
As discussed, have added the start up delay in the execution thread group itself.